src/script/pypackages.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+MGR_FOLDER=src/pybind/mgr
+MGR_REQUIREMENTS=$MGR_FOLDER/requirements-mgr.lock.txt
+
+export PYPACKAGES_FOLDER=$MGR_FOLDER/__pypackages__
+
+function install-pip-packages() { 
```

Comment:
At some we should refactor the install-deps stuff, and an alternative that I liked a lot was `spdk`'s [`pkgdep.sh`](https://github.com/spdk/spdk/blob/master/scripts/pkgdep.sh). That puts all [the distro-independent scripts in a `pkgdep/common.sh`](https://github.com/spdk/spdk/blob/master/scripts/pkgdep.sh#L119).